ResidentInformationApi.Tests/V1/Factories/ResponseFactoryTests.cs

```diff
@@ -0,0 +1,111 @@
+// using System.Collections.Generic;
```

Comment:
Should this be deleted if it is all commented out?

Reply:
The factories weren't in the scope of this ticket. I believe @louiseswift commented this out so it would not impact the tests we were running, but meant that teams working on other tickets would still have the base API code as a reference. I'll ask her to confirm when she's back.

Reply:
The production code for them is still here. I think either that should be commented out - or the tests should be uncommented. We don't want to be adding untested code.

Reply:
Have uncommented the tests and got them passing.